web/adventures/test_adventures.py

```diff
@@ -57,3 +58,33 @@ def test_URLListField_get_prep_value(self):
         """Test URLListField's to_python with none"""
         url_list = URLListField()
         self.assertEqual(url_list.get_prep_value(["www.google.com", "another.website.io"]), "www.google.com,another.website.io")
+
+
+class AdventureByIdTestCase(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+
+    def test_adventure_by_id_get_success(self):
+        test_adv = Adventure.objects.create(name="LMoP", links=["www.google.com", "another.website.io"])
+        lmop = self.client.get('/adventures/adventure/{}'.format(test_adv.id))
+        actual_data = json.loads(lmop.content.decode('utf-8'))
+        correct_data = {'name': 'LMoP',
+                        'id': 1,
+                        'links': ["www.google.com", "another.website.io"],
+                        'authors': [],
+                        'description': ''
+                        }
+        self.assertEqual(actual_data, correct_data)
+
+    def test_adventure_by_id_get_404(self):
+        notfound = self.client.get('/adventures/adventure/15')
```

Comment:
Instead of hard-coding urls you should use [`urlresolvers.reverse(viewname)`](https://docs.djangoproject.com/en/1.9/ref/urlresolvers/#reverse) with `args` or `kwargs` to avoid errors when URLs change.